### PR TITLE
Text underline moved to the back 

### LINKED ISSUE #703: 🐛 [Design Fix]: The line under the text hide the text
### Descrcibe the bug?

The underlined line hides the text  as you can see it in the attached screen shot
image

### Steps to reproduce the bug?

Go on to the home page of the reactpaly.io and scroll down to the What our community says section

### Expected behavior

The underline should be behind the text as shown in the attached screen shot.

image

This behavior is already implement in the footer as you can see in the attached screen shot.
image

### Desktop (Please provide your system information)

OS:Windows
browser: Chrome

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

Current behavior
image

Expected behavior
image

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Hat52**
# Description

The underline was hiding the text behind. Now it is moved to the back of the text. closes #703 

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

I have tested the changes by running them on my local.

# Checklist:

- [x] I have performed a self-review of my own code

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
@Hat52 , Can you add the proper title for the PR?

File: src/common/playcreated/play-created.scss [hunk 1]
```diff
@@ -21,7 +21,6 @@
       content: "";
       position: absolute;
       height: 4px;
-      background-color: var(--color-brand-primary);
```
Comment (Sachin-chaurasiya) [inline src/common/playcreated/play-created.scss:24]: @Hat52 , Why we are removing this?

Comment (Hat52) [reply src/common/playcreated/play-created.scss:24]: @Sachin-chaurasiya this is the change from other pull request that is why I this change appears here.

Comment (Hat52) [reply src/common/playcreated/play-created.scss:24]: removing this line gives us the desired result that I have described in the issue.

**Comment by Hat52** [comment]
@Sachin-chaurasiya Done

File: src/common/playcreated/play-created.scss (hunk 1 above)
Comment (Sachin-chaurasiya) [reply src/common/playcreated/play-created.scss:24]: @Hat52 , This is a common CSS that may affect the other places.

Comment (Hat52) [reply src/common/playcreated/play-created.scss:24]: @Sachin-chaurasiya I does't effect any other element. I have tested it.

Comment (Sachin-chaurasiya) [reply src/common/playcreated/play-created.scss:24]: @Hat52 , Still I would suggest solving this issue in some other way.

Comment (Hat52) [reply src/common/playcreated/play-created.scss:24]: @Sachin-chaurasiya okay I will look into it

Comment (Hat52) [reply src/common/playcreated/play-created.scss:24]: @Sachin-chaurasiya I have looked at the code and this class name is not a generic class Name it is only associated to those text that have a strong element and I have tested this fix and it does't break css anywhere.

Comment (Sachin-chaurasiya) [reply src/common/playcreated/play-created.scss:24]: Thanks, @Hat52 , for verifying it.

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by atapas — APPROVED** [review]
This is fixed.

**Pull request merged**